src/store/root-reducer.ts

```diff
@@ -0,0 +1,29 @@
+import persistCombineReducers from "redux-persist/es/persistCombineReducers";
+import persistReducer from "redux-persist/es/persistReducer";
+import storage from 'redux-persist/lib/storage'
+import habitReducer from '../features/habits/slice';
+import progressReducer from '../features/progress/slice';
+
+const habitPersistConfig  ={
+    key: 'habit', 
```

Comment:
It's preferred to keep sliced name as a constant in the slice file and re export and use it here